client/src/components/EventView/YoureInvited/index.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "styled-components";
+import PrimaryButton from "../../Common/Buttons/PrimaryButton";
+import SecondaryButton from "../../Common/Buttons/SecondaryButton";
+
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const CondensedMainHeader = styled.h1`
+  margin-block-end: .2em;
+`;
+
+const CondensedSecondaryHeader = styled.h3`
+  margin-block-start: .1em;
+  font-weight: normal;
+`;
+
+// I don't think we want to do it this way, I just wasn't aware of how it would be all built out afterwards.
+// This can be easily changed if need be.
+const ButtonWidth = styled.div`
```

Comment:
Instead of doing this, what you can do is style the buttons directly and remove the fullWidth attribute on the buttons. 

```
const StyledPrimary = styled(PrimaryButton)`
    width: [whatever width looks good];
`
```

You can do a similar thing for both the Primary and the Secondary button. You will also want to add some padding between the two buttons.